tests/foreman/api/test_setting.py

```diff
@@ -1,13 +1,13 @@
 # -*- encoding: utf-8 -*-
 """Test class for Setting Parameter values
 
-:Requirement: Setting
+:Requirement: Settings
```

Comment:
It should be `Settings` which is more appropriate because is it is not a singular, we can have multiple settings. This is even I can see Satellite UI as well.